integrations/ultralytics-yolov8/README.md

```diff
@@ -0,0 +1,315 @@
+          symmetric: False
+      model.19.act:
+        output_activations:
+          num_bits: 8
+          symmetric: False
+      model.21.cv1.act:
+        output_activations:
+          num_bits: 8
+          symmetric: false
+      model.21.m.0.cv1.conv:
+        input_activations: null
+      model.21.m.0.cv2.act:
+        output_activations:
+          num_bits: 8
+          symmetric: False
+      model.21.m.1.cv1.conv:
+        input_activations: null
+      model.21.m.1.cv2.act:
+        output_activations:
+          num_bits: 8
+          symmetric: False
+      model.22.cv2.0.0.conv:
+        input_activations: null
+      model.22.cv3.0.0.conv:
+        input_activations: null
+```
+</details>
+   
+The key `Modifiers` for sparse transfer learning are the following:
+- `ConstantPruningModifier` instructs SparseML to maintain the sparsity structure of the network during the fine-tuning process
+- `QuantizationModifier` tells SparseML to quantize the weights with quantization aware training over the last 5 epochs. The `scheme_overrides` are a bit complicated here to handle the GeLU activations followed by the C2f module introduced in YOLOv8, but you do not need to worry too much about this.
+
+SparseML parses the `Modifers` in the recipe and updates the training loop with logic encoded therein.
+
+Run the following to transfer learn from the 80% pruned YOLOv8m onto the COCO128 dataset:
+```bash
+sparseml.ultralytics.train \
+  --model "zoo:cv/detection/yolov8-m/pytorch/ultralytics/coco/pruned80-none" \
+  --recipe "zoo:cv/detection/yolov8-m/pytorch/ultralytics/voc/pruned80_quant-none" \
+  --data "coco128.yaml" \
+  --batch 2
+```
+
+The script uses the SparseZoo stubs to identify and download the starting checkpoint and YAML-based recipe file from the SparseZoo. SparseML parses the transfer learning recipe and adjusts the training process to maintain sparsity during the fine-tuning process. It then kicks off the training process. Note that in the command above, the `--model` has a zoo stub indicating the model was trained on COCO whereas the `--recipe` has a zoo stub indicating the model was trained on VOC. As such, SparseML downloads the 80% pruned model trained on COCO and the sparse transfer learning recipe that was used to fine-tune and quantized the model on VOC.
+
+The resulting model is 80% pruned and quantized and is trained on COCO128!
+
+Of course, you should feel free to download the recipes and edit the hyperparameters as needed!
+
+### **Export to ONNX**
+
+Run the `sparseml.ultralytics.export_onnx` command to export the model to ONNX. Be sure the `--model` argument points to your trained model.
+
+```bash
+sparseml.ultralytics.export_onnx \
+  --model ./runs/detect/train/weights/last.pt \
+  --save_dir yolov8-m
+```
+
+### **DeepSparse Deployment**
+
+Once exported to ONNX, you can deploy your models with DeepSparse. Checkout the [DeepSparse Repository](https://github.com/neuralmagic/deepsparse) for more details.
```

Comment:
Maybe we should point to https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/yolov8 to be more specific